app/routes/__frontend/resources/developer-resources/open-source/about-open-source.mdx

```diff
@@ -30,62 +31,85 @@ In this section, we will talk about all open source!
 
 ## What is Open Source Software (OSS)?
 
-Open Source Software (OSS) is software where the source code is generally made freely available and accessible to the public. It is distributed under a copyright license that permits reuse, modification, and redistribution and encourages open collaboration and peer production.
+Open Source Software (OSS) is software where the source code is freely available and accessible to the public. It is distributed under a copyright license that permits reuse, modification, and redistribution and encourages open collaboration and peer production.
 
 There are various licenses in open source. Some grant almost complete freedom, including incorporating code into commercial, closed-source projects. While in contrast, others may be more restrictive.
 
-There is a culture in open source of building off of each other's work. You can take an open source project and extend it or add it to another project as long as you comply with the terms of the license. Package managers—such as NPM—can also incorporate other open source code into your project.
+There is a culture in open source of building off of each other's work. You can take an open source project and extend it or add it to another project as long as you comply with the terms of the license. Package managers like NPM can also incorporate other open source code into your project.
 
 The ecosystem of open source software is vast, ranging from GUI programs through libraries and modules designed to provide specific functionality to other programs, frameworks, and programming languages, to platform infrastructure behind decentralized social networks, blockchain protocols, and simple personal projects.
 
 Although best known in software, open source concepts can extend to any field with intellectual property, such as hardware development, design systems, and designs for 3D printable goods.
 
----
+</TextContainer>
 
-## Building your portfolio with open source
+<TextContainer>
+
+## Building a Portfolio with Open Source
 
 Doing open source work is a great way to improve your skills, build your portfolio, and get noticed by others in the industry. And this is one of the reasons why you want to get involved in and contribute to open source.
 
-Many employers value open source work in tech. So you should make space for your contributions list on your resume, especially if you are looking for your first developer job. It will help you stand out from other candidates.
+Many employers value open source work in tech. Make space for your contributions list on your resume, especially if you are looking for your first developer job. It will help you stand out from other candidates.
```

Comment:
Many employers value open source work because it requires skills you'll use while working a tech job. Make space for your contributions list on your resume, especially if you are looking for your first developer job. It will help you stand out from other candidates.